opengrok-web/src/main/webapp/js/utils-0.0.43.js

```diff
@@ -1484,6 +1484,7 @@ function init_markdown_converter() {
                         converter.setOption('tasklists', true);
                         converter.setOption('simplifiedAutoLink', true);
                         converter.setOption('parseImgDimension', true);
+                        converter.setOption('literalMidWordUnderscores', true);
```

Comment:
You need to increment the file version to `0.0.44.js` and update `Scripts.java` (look at https://github.com/oracle/opengrok/pull/3717/files for reference). The rename is needed due to caching policies.